QuasistaticBrownianThermalNoise/QuasistaticBrownianThermalNoise.cpp

```diff
@@ -1202,6 +1368,10 @@ CylTransFunc::CylTransFunc(double coatThick, double halfCylThick):
     double local_coatingEnergy = 0.;
     double local_substrateEnergy = 0.;
     double local_thisEnergy = 0.;
+      
+    double local_U_coating_energy = 0;
+    double local_U_this_energy = 0.;
+      
     typename DoFHandler<dim>::active_cell_iterator 
```

Comment:
why new variables here? Do you use local_coatingEnergy and local_U_coating_energy? How are they different?